tools/accuracy_checker/openvino/tools/accuracy_checker/annotation_converters/cvat_hands_and_palm.py

```diff
@@ -0,0 +1,207 @@
+            bbox = bbs[0]['bbox']
+
+            identifier = image['file_name'].split('/')[-1]
+            if check_content:
+                if not check_file_existence(self.images_dir / identifier):
+                    content_errors.append('{}: does not exist'.format(self.images_dir / identifier))
+            landmarks_x, landmarks_y = self.get_landmarks(keypoints, num_landmarks)
+            landmarks_annotation = HandLandmarksAnnotation(identifier, landmarks_x, landmarks_y)
+            if self.from_landmarks:
+                landmarks_annotation.metadata['rect'] = [np.min(landmarks_x) - self.padding,
+                                                         np.min(landmarks_y) - self.padding,
+                                                         np.max(landmarks_x) + self.padding,
+                                                         np.max(landmarks_y) + self.padding]
+            else:
+                landmarks_annotation.metadata['rect'] = [bbox[0], bbox[1], bbox[0] + bbox[2], bbox[1] + bbox[3]]
+            annotations.append(landmarks_annotation)
+            if progress_callback is not None and image_id % progress_interval == 0:
+                progress_callback(image_id * 100 / len(annotation['images']))
+
+        return ConverterReturn(annotations, meta, content_errors)
+
+
+class CVATPalmDetectionConverter(FileBasedAnnotationConverter, CVATHandPalmConverterBase):
+    __provider__ = 'cvat_palm_detection'
+    annotation_types = (DetectionAnnotation, )
+
+    @classmethod
+    def parameters(cls):
+        configuration_parameters = super().parameters()
+        configuration_parameters.update({
+            'images_dir': PathField(
+                is_directory=True, optional=True,
+                description='path to dataset images, used only for content existence check'
+            ),
+            'landmarks_file': PathField(
+                is_directory=False, optional=True,
+                description='path to file with hand landmarks data'
+            ),
+            'padding': NumberField(optional=True, default=40, value_type=int,
+                                   description='additional padding while acquiring bounding box data from landmarks'
+                                   ),
+        })
+        return configuration_parameters
+
+    def configure(self):
+        super().configure()
+        self.images_dir = self.get_value_from_config('images_dir') or self.annotation_file.parent
+        self.landmarks_file = self.get_value_from_config('landmarks_file')
+        self.padding = self.get_value_from_config('padding')
+
+    def convert(self, check_content=False, progress_callback=None, progress_interval=100, **kwargs):
+        if self.landmarks_file is not None:
+            return self.convert_from_landmarks(check_content, progress_callback, progress_interval, **kwargs)
+
+        annotation = read_json(self.annotation_file)
+        meta = {'label_map': {0: 'PALM'}, 'label': ['PALM']}
```

Comment:
please define and use get_meta method for obtain meta

Reply:
fixed